indico/MaKaC/conference.py

```diff
@@ -3582,18 +3572,14 @@ def canAccess( self, aw ):
         if has_request_context() and self.__ac.isHarvesterIP(request.remote_addr):
             return True
 
-        # Managers have always access
-        if self.canModify(aw):
-            return True
-
```

Comment:
Moved it down to avoid calling `canModify` in case the event is public and has no IP restriction (no need to trigger SQL queries from canModify/can_manage in that case)